.ci/windows-steps.yaml

```diff
@@ -24,8 +24,7 @@ steps:
 - bash: |
     git clone --depth 1 https://github.com/mlpack/jenkins-conf.git conf
 
-    curl -O -L https://sourceforge.net/projects/arma/files/armadillo-9.800.6.tar.xz -o armadillo-9.800.6.tar.xz
-    tar -xzvf armadillo-9.800.6.tar.xz
+    curl -L https://sourceforge.net/projects/arma/files/armadillo-9.800.6.tar.xz | tar -xvJ
 
     cd armadillo-9.800.6/ && cmake $(CMakeGenerator) \
     -DBLAS_LIBRARY:FILEPATH=$(Agent.ToolsDirectory)/OpenBLAS.0.2.14.1/lib/native/lib/x64/libopenblas.dll.a \
```

Comment:
OpenBLAS 0.2.14 is pretty ancient at this point, with many known bugs.

Reply:
Agreed, it's the latest version that you get on nuget (https://www.nuget.org/packages/OpenBLAS/), @shrit what version does the auto-downloader use?

We could push a new version to nuget, or maybe use the blas lib that is bundled with armadillo?

Reply:
I think the version that is bundled with Armadillo is just reference BLAS/LAPACK; it would probably be better to test with OpenBLAS, which is what more people are using.  At least personally, just for the CI build, I don't think it's too huge of an issue to test against an old version of OpenBLAS, so long as we aren't seeing test failures in our own code because of it (I don't think we are).

Of course there is no harm in upgrading OpenBLAS either.  It seems like the OpenBLAS nuget package maintainer was last seen six or seven years ago, so, it might be a little tricky to get a new version pushed. :)

Reply:
Agreed for the CI, this is good for now. But we should still see if we can update the nuget package, because we tell people to use the nuget package as well. I'll see if I can get a response from the OpenBLAS nuget maintainer. On the other hand it packages just a simple `lib/dll` so we could host that as well and tell people to download that.

Reply:
> I think the version that is bundled with Armadillo is just reference BLAS/LAPACK

As of Armadillo 10.x, OpenBLAS is bundled:  https://gitlab.com/conradsnicta/armadillo-code/-/tree/10.6.x/examples/lib_win64

Reply:
@zoq The autodownloader is using a recent version of OpenBLAS: 0.3.13

If you would like, our ensmallen nuget package does contain a version of OpenBLAS inside, it might be possible to point out directly to it instead of installing from the OpenBLAS  official repo.
But we need to know first what is happening there that is causing armadillo issue :smile: